Mathlib/Topology/Instances/ENNReal/Lemmas.lean

```diff
@@ -619,10 +621,6 @@ protected theorem tsum_add : ∑' a, (f a + g a) = ∑' a, f a + ∑' a, g a :=
 protected theorem tsum_le_tsum (h : ∀ a, f a ≤ g a) : ∑' a, f a ≤ ∑' a, g a :=
   ENNReal.summable.tsum_le_tsum h ENNReal.summable
 
-@[gcongr]
```

Comment:
Is this subsumed by other changes now (the `gcongr`-to_additive change? something else?)?